python/python/tests/tests_server_modules/test_ft.py

```diff
@@ -198,3 +199,85 @@ async def _create_test_index_with_vector_field(
             schema=fields,
             options=FtCreateOptions(DataType.JSON, prefixes=prefixes),
         )
+
+    @pytest.mark.parametrize("cluster_mode", [True])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_ft_explain(self, glide_client: GlideClusterClient):
+        indexName = str(uuid.uuid4())
+        await TestFt._create_test_index_for_ft_explain_commands(
+            self=self, glide_client=glide_client, index_name=indexName
+        )
+
+        # FT.EXPLAIN on a search query containing numeric field.
+        query = "@price:[0 10]"
+        result = await ft.explain(glide_client, indexName=indexName, query=query)
+        assert b"Field {\n  price\n  0\n  10\n}\n" == result
```

Comment:
I afraid that parsing may change in future or produce less whitespaces. Maybe we don't need to check the response value (we are not testing the module).

Reply:
Updated.